src/goom_config.h

```diff
@@ -2,10 +2,10 @@
 #define GOOM_CONFIG
 
 #ifdef MID
-#define GOOM_TEXTURE_WIDTH  256
+#define GOOM_TEXTURE_WIDTH  455
```

Comment:
It can be changed, but must be something in 16/9
That's also the reason why I changed it, the ring before was not round :-D